tethys.Utils.exceptions.DataError
=================================

.. currentmodule:: tethys.Utils.exceptions

.. autoexception:: DataError